extensions/transform/org.eclipse.smarthome.transform.exec/README.md

```diff
@@ -6,14 +6,44 @@ The external program must either be in the executable search path of the server
 
 ## Example
 
-With the command line 
+**.items**
 
+This will replace the visible label in the UI with the transformation you apply with the command <YourCommand>.
+  
+```java
+String yourItem "Some info  [EXEC(/absolute/path/to/your/TransformProgram %s):]"
 ```
+  
+**.thing**
+
+This example shows the transformation of the output channel of an item.
+
+```java
+Thing yourThing [ ... , transform="EXEC(/absolute/path/to/your/TransformProgram %s)"]
+```
+
+**.rules**
+
+```java
+rule "Your Rule Name"
+  when
```

Comment:
It does not really matter, but I assume you can remove the space in front of `when`.

If `java` used as preformatted text interpretation by intention? Why not `text`?

Reply:
Because it was demanded so from the openhab team.